recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "occt" %}
-{% set version = "7.9.0" %}
-{% set build = 4 %}
+{% set version = "7_9_1" %}
```

Comment:
so this kinda broke the version identification.

I suggest you all go back to "7.9.1" and use a filter to get the right version.

You should also mark 7_9_1 package as broken, since conda thinks it is 791

See the results in:
https://anaconda.org/conda-forge/occt

Reply:
moving to rattler-build in #132 s.t. this won't pass unnoticed anymore